### ISSUE TITLE
Suggestion RE: Tree planting

### ISSUE THREAD

**Issue by oholsuggestions** [jasonrohrer/OneLife#884]
Problem: We are unable to plant Willow trees and Bald Cypress trees. These trees do not have a unique droppings, however for decorative purposes it would be lovely if we can plant them as well.

If there is a preference for unique droppings I would suggest dropping two logs for the Bald Cyprus trees (costing double the amount of hungry work), possibly without the additional firewood. 

I also found[ information](https://uphomely.com/is-willow-wood-good-for-anything/) on how Willow bark has traditionally been used to reduce fevers.  Perhaps we can use that to cure yellow fever quickly? We could mash up its leaves and boil it on coal to make a strong , fever relieving tea.

**Comment by QuirkySmirkyIan** [comment]
Hungry work is 10 pips and can't be used without a few extra pips of reserve. Even with most yum diets you basically cannot cut trees for the last few minutes of your life. I do not think 20 pip hungry work would be a welcomed changed with the current system. However, I believe the player base would welcome 20 pip hungry work for the bird trees which are considered SOFTWOOD but are one of the hardest items to make.

**Comment by oholsuggestions** [comment]
> However, I believe the player base would welcome 20 pip hungry work for the bird trees which are considered SOFTWOOD but are one of the hardest items to make.

Yes! I didn't even think of the cardinal tree. It takes quite some time to make, so to have the ability of easily chopping it down sucks.

**Comment by TwistedHBG** [comment]
This belongs in https://github.com/jasonrohrer/OneLifeData7/issues

**Comment by jasonrohrer** [comment]
With the bald cypress at least... these are hardcore swamp trees.  I could let you grow them in swamp only, but that would diminish their usefulness for decorative purposes.

Willow is a little bit more flexible.  I will think about it.
